components/build_image.py

```diff
@@ -75,24 +80,28 @@ def wait_for_docker_daemon(timeout=60):
   # TODO(lunkai): use TimeoutError when we use py3.
   raise RuntimeError
 
+
 def get_build_args(config):
   """
-  Make the list of params for docker build from config.
+    Make the list of params for docker build from config.
```

Comment:
Is it normal for the comment to be indented with respect to """?

Reply:
Nice catch, this is not normal, it'll be fixed in next commit.